draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -285,29 +285,24 @@ The old owner may remove the member zone containing the `coo` property from `$OL
 With a `group` property, consumer(s) can be signaled to treat some member zones within the catalog zone differently.
 
 The consumer MAY apply different configuration options when processing member zones, based on the value of the `group` property.
-The exact handling of configuration referred to by the `group` property value is left to the consumer's implementation and configuration.
-The property is defined by a TXT record in the sub-node labeled `group`.
+A `group` property value is stored as content of a TXT record directly below the member node.
+The exact handling of the `group` property value is left to the consumer's implementation and configuration.
 
 The producer MAY assign a `group` property to all, some, or none of the member zones within a catalog zone.
 The producer MAY assign more than one `group` property to one member zone. This will make it possible to transfer group information for different consumer operators in a single catalog zone.
 Consumer operators SHOULD namespace their group property values to limit risk of clashes.
```

Comment:
```suggestion
Implementations MAY facilitate mapping of a specific `group` value to specific configuration configurable *on a per catalog zone basis* to allow for producers that publish their catalog zone at multiple consumer operators.
Consumer operators SHOULD namespace their group property values to limit risk of clashes.
```

@mind04 Adding this (adapted) suggestion again because the previous one was outdated because of follow-up commits.